vocode/streaming/utils/mp3_helper.py

```diff
@@ -0,0 +1,23 @@
+import io
+import wave
+import miniaudio
+from vocode.streaming.models.audio_encoding import AudioEncoding
+from vocode.streaming.utils import convert_wav
+
+# sampling_rate is the rate of the input, not expected output
+def decode_mp3(mp3_bytes: bytes) -> io.BytesIO:
+    mp3_chunk = io.BytesIO(mp3_bytes)
+
+    # Convert it to a wav chunk using miniaudio
+    wav_chunk = miniaudio.decode(mp3_chunk.read(), nchannels=1)
```

Comment:
how come we need to make an `mp3_chunk` in memory file if we're just going to `read` from it?